apps/crm/src/modules/mailchimp/config-sections/instructions/instructions.tsx

```diff
@@ -0,0 +1,94 @@
+import { Section } from "../../../ui/section/section";
+import React from "react";
+import { List, Text, TextProps, PropsWithBox, Box, Button, Chip } from "@saleor/macaw-ui/next";
+import { useLocalStorage } from "usehooks-ts";
+import { TextLink } from "../../../ui/text-link/text-link";
+
+const P = (props: TextProps) => <Text marginBottom={5} as="p" {...props} />;
+
+const H = (props: TextProps) => (
+  <Text as="h2" variant="heading" marginBottom={4} marginTop={12} {...props} />
+);
```

Comment:
Nit: Something rubs me the wrong way about it. Why not `Paragraph` and `Heading`?